pkg/client/defaults.go

```diff
@@ -51,7 +52,7 @@ func NewDeleteConfig() *DeleteConfig {
 // NewLogConfig is a LogConfig with follow disabled and default images.
 func NewLogConfig() *LogConfig {
 	return &LogConfig{
-		Follow:    false,
-		Namespace: config.DefaultNamespace,
```

Comment:
Since we are leaving in the namespace defaulting in the other methods, we should do the same here, right? Was there some other reason to specifically pull out namespace from this and the delete config?